nvflare/private/fed/server/server_command_agent.py

```diff
@@ -43,34 +43,40 @@ def start(self, engine):
 
     def listen_command(self, engine):
         try:
```

Comment:
this code seems to be duplicate as above, consider wrap into a function ?

Reply:
done